cfpb/cfgov-refresh
# Fix missing margins underneath rendered rich text fields

We lost a bunch of margins when our new `parse_links` middleware started rendering rich text fields with `div.rich-text` wrappers. This PR fixes the issue (https://GHE/CFGOV/platform/issues/2983) with CSS.

## Additions

- New `rich-text.less` file for Less used to mitigate the markup generated by Wagtail's rich text field.

## Removals

- Some Less that zeroed out bottom margin for `:last-child`ren, which has been moved to cf-core

## Testing

1. Load a page like http://localhost:8000/consumer-tools/military-financial-lifecycle/
1. Observe the lack of margin between the paragraph and the link in the Featured Content Module, and between the video in the sidebar and the first link under it
1. Pull this branch
1. `gulp styles`
1. Reload the page and see the margin restored

## Screenshots

### Before

![screen shot 2018-09-19 at 16 48 00](https://user-images.githubusercontent.com/1044670/45780603-e9631300-bc2b-11e8-89c2-107f39da6a70.png)

### After

(missing image is expected in local environment; just look at the space between the paragraph and the link)

![screen shot 2018-09-19 at 16 48 56](https://user-images.githubusercontent.com/1044670/45780674-1283a380-bc2c-11e8-9f16-2022ab51e820.png)

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
